openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -32,10 +52,10 @@ def __init__(self):
         self.supports['inequality_constraints'] = True
         self.supports['equality_constraints'] = True
         self.supports['multiple_objectives'] = False
-        self.supports['two_sided_constraints'] = True
 
         # TODO: Support these
         self.supports['linear_constraints'] = False
+        self.supports['two_sided_constraints'] = False
```

Comment:
ha, OK I did not regenerate all the docstrings when I merged out.  any others I need to worry about?